### DISCUSSION TITLE
How to create GLMakie window in full screen mode?

### DISCUSSION THREAD

**Discussion by prog900** [JuliaPlots/Makie.jl#1660] [Q&A] [answered]
"fig = GLMakie.Figure(resolution=(960,650))"

Is there a parameter to immediately create a window in the full screen mode?

**Comment by mauera** [comment, marked as answer]
Not that I'm aware of, but one idea is to pass in the exact dimensions  of your display.

**Reply by asinghvi17** [reply]
Pretty much - if you are on Windows or Mac, you can call `Makie.primary_resolution() to automatically get the resolution of your primary display, and set your resolution to that.
